src/compiler/utilitiesPublic.ts

```diff
@@ -1928,8 +1931,8 @@ export function isPropertyAccessOrQualifiedName(node: Node): node is PropertyAcc
 }
 
 /** @internal */
-export function isCallLikeOrFunctionLikeExpression(node: Node): node is CallLikeExpression | SignatureDeclaration {

```

Comment:
Was this function really not used elsewhere? Super surprised, but it does seem to be true.